lib/ex338/coherence/rememberable.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Ex338.Coherence.Rememberable do
+  @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
+  import Ecto.Query
```

Comment:
Most of the time you are using the multi-alias/require/import/use syntax, but here you are using multiple single directives

Reply:
ok